language/stdlib/modules/LibraSystem.move

```diff
@@ -63,23 +63,19 @@ module LibraSystem {
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    // Methods operating the Validator Set config
-    // callable by Validator's operators
+    // Methods operating the Validator Set config callable by the Association only
     ///////////////////////////////////////////////////////////////////////////
 
     // Adds a new validator, this validator should met the validity conditions
     public fun add_validator(
-        operator: &signer,
+        assoc: &signer,
         account_address: address
     ) acquires CapabilityHolder {
-        // Validator's operator can add its certified validator to the validator set
-        assert(
-            Signer::address_of(operator) == ValidatorConfig::get_operator(account_address),
-            22
-        );
+        // Only the Association may add a validator to the validator set
+        assert(Signer::address_of(assoc) == CoreAddresses::ASSOCIATION_ROOT_ADDRESS(), 1002);
```

Comment:
Instead of doing a check on the `assoc` address, could you instead pass in a `_: &Capability<AssociationRootRole>` here? e.g.
```rust
public fun add_validator(
   _: &Capability<AssociationRootRole>,
   account_address: address
) ... {
// Can remove line 75, no checks are needed since the capability was passed in.
....
}
```